presentation/components/src/presentation-components/tree/controlled/PresentationTreeNodeRenderer.scss

```diff
@@ -15,14 +15,12 @@
   .presentation-components-filter-action-button {
     visibility: hidden;
     position: sticky;
-    height: 24px;
-    width: 24px;
     background-color: inherit;
     right: 0;
 
     .iui-button{
-      width: 100%;
-      height: 100%;
+      height: 24px;
+      width: 24px;
       display: flex;
```

Comment:
`display: flex` should be on `.presentation-components-filter-action-button` div instead.